api/logging/__init__.py

```diff
@@ -19,7 +21,30 @@ def parse_log_entry(_logger, entry):
 
     return lowercase_level, message, extra
 
+
 def push_log_entry(_logger, level, message, extra):
     """ Logs a single log entry at the specified level """
     logging_fun = getattr(_logger, level, None)
-    logging_fun(message, extra=extra)
\ No newline at end of file
+    logging_fun(message, extra=extra)
+
+
+class RequestResponseLogging:
+    def __init__(self, logger, app: Flask = None, no_log_paths=['/logs']):
```

Comment:
suggestion: `urls_deny_list` is a more familiar name for the behavior needed

Reply:
Nice, thanks. I struggled to find a decent name